ui/DesignSystem/Component/EmptyState.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import Icon from "./EmptyState/Icon.svelte";
+  import { Button, Text } from "../Primitive";
+
+  export let icon = "tent";
+  export let text = "Nothing to see here";
+  export let mainCtaText = "Go back";
+  export let mainDataCy = null;
+  export let mainCtaAction = null;
```

Comment:
The idiomatic Svelte way of doing callbacks is to define custom events for components: https://svelte.dev/examples#component-events.

Here's an example in our own codebase on how to do it:
https://github.com/radicle-dev/radicle-upstream/blob/master/ui/DesignSystem/Component/SegmentedControl.svelte

Reply:
addressed in [ed0bd95](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/ed0bd952a76832c19aa7cdd6c8d6f783d0cc78d3)